lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -121,6 +121,39 @@ def ks_console
             (@port && @baud) ? "console=ttyS#{@port},#{@baud}" : ''
           end
 
+          apipie :method, 'Generates commands for package(s) installation, supported OS families: Redhat, Debian & Suse' do
+            required :packages, String, desc: "Package(s) to install"
+            returns String, desc: 'Installation commands'
+            example "CentOS host: install_packages('pkg1') #=> yum -y install pkg1"
+            example "Fedora host: install_packages('pkg1 pkg2') #=> dnf -y install pkg1 pkg2"
+          end
+          def install_packages(packages)
+            raise UndefinedPackages.new if packages.empty?
+
+            case host.operatingsystem&.family
+            when 'Redhat'
+              if host.operatingsystem.name.downcase == 'fedora'
```

Comment:
Technically on EL8+ DNF is also the package manager but yum is a symlink to dnf. I wonder if EL9 will still provide the compatibility symlink (wouldn't surprise me), but it's something to consider.

Reply:
:+1: for using dnf on el8+ (ideally also for centos stream which we today recognize still as centos)

Reply:
For both managers commands are same (`dnf/yum install -y pkg`), so not sure if there is need to have special condition for EL8+, or just keep it as it is.

Reply:
The reasoning I'd have is that I'd want to be prepared for the future. If EL9 decides that `dnf` is the tool and drops `yum` then we won't have to make changes to support EL9. History has also taught us that a release is always closer than you think. It may be good to reach out internally to see what is the preferred tool.